### PR TITLE
Wayland pointer frame

### PR DESCRIPTION
**Pull request by akallabeth**

### CODE REVIEW

File: winpr/libwinpr/utils/wlog/ConsoleAppender.c [hunk 1]
```diff
@@ -138,6 +138,7 @@ static BOOL WLog_ConsoleAppender_WriteMessage(wLog* log, wLogAppender* appender,
 
 	if (message->Level != WLOG_OFF)
 		fprintf(fp, "%s%s\n", message->PrefixString, message->TextString);
+	fflush(fp);
```
Comment (hardening) [inline winpr/libwinpr/utils/wlog/ConsoleAppender.c:141]: Hum this one seems unrelated.
Doesn't it have the possibility to kill the performances of the console appender ?

Comment (akallabeth) [reply winpr/libwinpr/utils/wlog/ConsoleAppender.c:141]: oops ^^

**Comment by akallabeth** [comment]
@hardening ok, cleaned up the commits, now there should be no debug leftovers.
